plonky2x/src/frontend/merkle/tendermint.rs

```diff
@@ -161,17 +147,16 @@ impl<L: PlonkParameters<D>, const D: usize> CircuitBuilder<L, D> {
         current_nodes[0]
     }
 
-    // TODO: Compute NB_LEAVES from NB_ENABLED_LEAVES.
-    pub fn compute_root_from_leaves<
-        const NB_ENABLED_LEAVES: usize,
-        const NB_LEAVES: usize,
-        const LEAF_SIZE_BYTES: usize,
-    >(
+    pub fn compute_root_from_leaves<const NB_LEAVES: usize, const LEAF_SIZE_BYTES: usize>(
```

Comment:
Should we remove the `NB_LEAVES` const generic here?

Reply:
Keeping this, updating input types to `ArrayVariable`.